.github/workflows/ci-comment-failures.yml

```
@@ -117,7 +117,7 @@ jobs:
                 owner: context.repo.owner,
                 repo: context.repo.repo,
                 issue_number: issue_number,
-                body: 'You added a new example but didn\'t update the readme. Please run `cargo run -p build-templated-pages -- update examples` to update it, and commit the file change.'
+                body: 'The examples readme needs to be updated. Please run `cargo run -p build-templated-pages -- update examples` to update it, and commit the file change.'
```

I'd prefer we give a "why", even if vague. It's always frustrating to be told off without being given a reason. Automation makes it worse.
```suggestion
                body: 'Because the examples' description changed, the examples README.md needs to be updated. Please run `cargo run -p build-templated-pages -- update examples` to update it, and commit the file change.'
```

---

There are several possible reasons a contributor may see this message:

- They added an example or examples and didn't generate the readme
- They removed an example or examples and didn't generate the readme
- They modified an example's metadata and didn't generate the readme
- They modified the readme template and didn't generate the readme
- They modified the generated readme file when they needed to modify the example metadata instead.
- They modified the generated readme file when they needed to modify the template instead.

I went with this vague language because we can't know what they were trying to do. In the example I linked to in the PR description, the contributor modified the readme template. So the point of this PR is to not tell them that they changed an example description when they did not.

---

That feels a bit fluffy to me since the contributor knows that they made a PR that changes Bevy's code and this messages is coming to them in response to that.

What about something like

> The generated `examples/README.md` is out of sync with the example metadata in `Cargo.toml` or its template. Please run cargo run -p build-templated-pages -- update examples to update it, and commit the file change.